casanovo/config.py

```
@@ -71,7 +119,7 @@ class Config:
         log_metrics=bool,
         log_every_n_steps=int,
         lance_dir=str,
-        val_check_interval=int,
+        val_check_interval=lambda v: _coerce_val_check_interval(v),
```

Ruff (0.15.12)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)